.travis.yml

```diff
@@ -21,5 +21,5 @@ deploy:
   script:    
     - heroku container:release web --app $HEROKU_APP
   on:
-    branch: master
+    branch: module10
```

Comment:
Remember to change this back when you merge 🙂

Reply:
Done